lib/parsers/csv_layout/location_parser.rb

```diff
@@ -13,21 +13,18 @@ def validations
 
       def initialize(line, parser)
         @parser = parser
-        _parse(line)
-
-        valid?
+        parse(line)
       end
 
-      protected
+      private
 
-      def _parse(line)
+      def parse(line)
         begin
           @location = TokenUtil.pad_location(line[0].strip)
         rescue StandardError => e
-          errors.add(:location, 'There was an error while parsing the location')
```

Comment:
These errors were ending up in the void, as they can't survive a`valid?` We'll still get an invalid location error though, as the validator will be unable to find a location.